qt/scientific_interfaces/General/UserInputValidator.cpp

```diff
@@ -38,7 +38,7 @@ UserInputValidator::UserInputValidator() : m_errorMessages() {}
 bool UserInputValidator::checkFieldIsNotEmpty(const QString &name,
                                               QLineEdit *field,
                                               QLabel *errorLabel) {
-  if (field->text() == "") {
+  if (Mantid::Kernel::Strings::isEmpty(field->text().toStdString())) {
```

Comment:
Perhaps use Qt functionality when it's available and we're already in Qt land?:`text().trimmed().isEmpty()`?

Reply:
Yeah that looks like a better alternative I'll get that updated now. Didn't think of that at the time.